data/core/macros/abilities.cfg

```diff
@@ -75,6 +75,19 @@ This ability will not cure an affected unit of poison, however, only delay its e
     [/regenerate]
 #enddef
 
+#define ABILITY_SELF_HEAL
```

Comment:
I would like macro name be closer to ability id.

Reply:
Should the ID be the same as the Regen +8 ability, like how Healing +8 and +4 have the same `healing` ID?

Reply:
We didn't know what was best to do for the macro name. We considered "LESSER_REGENERATES" such as is the trend with heal and extra heal. But weren't sure that was actually better.

Also strangely the id for heals were both the same (as vultraz has pointed out above) so we weren't sure if that was standard procedure or if it would be better to make a new id. Please feel free to commit any change you think is appropriate.

Reply:
I believe the reason for same id is to not have them cumulative.